index.js

```diff
@@ -41,7 +53,21 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  if(movies.length === 0){
+    return 0;
+  }
+
+  let highestScore = 0;
+  for(movie of movies){
+    let metascore = parseInt(movie.metascore)
+    if(highestScore < metascore){
+      highestScore = metascore
+    }
+  }
+
+  return highestScore;
```

Comment:
Nice work.  Remember to add `;` at the end of our statements it helps the eye know when something is done.  Also start to spread out your `{` after closing parenthesis.  Last thing I'd remove the whitespace in this function.